File: server/dotnet/sdk-dotnet-example/Presentation/Models/CreatePaymentLinks/CreatePaymentLinkResponse.cs
using Business.Domain.Common.Enums;
using Business.Domain.Enums.PaymentLinks;

namespace Presentation.Models.CreatePaymentLinks;

public class CreatePaymentLinkResponse
{
    public string? PaymentLinkId { get; init; }
    
    public string? RedirectUrl { get; init; }
    
    public Status? Status { get; init; }
    
    public decimal? Amount { get; init; }
    
    public Currency? Currency { get; init; }
}

File: server/nodejs/src/presentation/models/payment-link/create-payment-link-response.ts
import { Status } from '../../../business/domain/common/enums/status';
import { Currency } from '../../../business/domain/common/enums/currency';

export interface CreatePaymentLinkResponse {
    paymentLinkId?: string | null;
    redirectUrl?: string | null;
    status?: Status | null;
    amount?: number | null;
    currency?: Currency | null;
}
